hack/update-generated-completions.sh

```diff
@@ -4,9 +4,6 @@
 
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
-# ensure we have the latest compiled binaries
-"${OS_ROOT}/hack/build-go.sh"
```

Comment:
It takes 1minute. We do not do this in _any_ script and this one doesn't get to be special.